### PR TITLE
Network: implement 0.4 marked compatibility removals

### PR DESCRIPTION
**Pull request by Berserker66**
These changes should all have been documented at least a month ago. People testing their clients would be appreciated.  The python clients should all be fine, even 0.3.3 clients worked in my testing, though they will say "No player connected". - But again, additional testing won't hurt.

### CODE REVIEW

**Comment by black-sliver** [comment]
apclientpp
* games are a set since version 0.2.6; never used the index
* exclusions since 0.2.6
* items_handling since 0.2.0
* never supported datapackage_version (the single int)

**Comment by black-sliver** [comment]
apcpp
* does not seem to use RoomInfo["games"]
* items_handling is being sent since 0.2.2
* datapackage_version would become always 0, but caching is currently not implemented afaik
* uses neither exclusions nor games in GetDataPackage

File: worlds/ff1/__init__.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ class FF1World(World):
     option_definitions = ff1_options
     game = "Final Fantasy"
     topology_present = False
+
     remote_items = True
```
Comment (Berserker66) [inline worlds/ff1/__init__.py:34]:
```suggestion
```

File: worlds/ff1/__init__.py [hunk 2]
```diff
@@ -30,7 +30,6 @@ class FF1World(World):
     option_definitions = ff1_options
     game = "Final Fantasy"
     topology_present = False
-    remote_items = True
     data_version = 2
     remote_start_inventory = True
```
Comment (Berserker66) [inline worlds/ff1/__init__.py:34]:
```suggestion
```

**Review by black-sliver — APPROVED** [review]

**Pull request merged**